Adding explanatory text to _infographics/climate/climate-change-scheme.md

willemfrishert opened the pull request:
Pilot adding English explanatory texts: #9 

### What does this PR do?

- bulk-translated the Czech text to English using [Google Translate](https://translate.google.com/) for climate-change-scheme.md
- Proofread the English text and correct it to sound natural

### Open questions:
1. The [Czech page](https://faktaoklimatu.cz/infografiky/schema-klimaticke-zmeny) has 3 preview boxes. I could only locate one of the 3 preview boxes.
- I managed to link the English page to "slug=fossil-fuels-emissions" for the first preview
- The second preview links to "slug=cykly-koncentrace-co2" and there does not seem to be an equivalent for English (yet)?
- The third preview links to the explainer "slug=vymirani-koralovych-utesu". There are no explainers in English (yet)?

What should we do with the second and third preview?
2. The [Czech page](https://faktaoklimatu.cz/infografiky/schema-klimaticke-zmeny) points to most English Wikipedia pages except for the one on the green house effect. I redirected that one to the English Wikipedia article to provide the reader with a primer.

willemfrishert committed:
Update _infographics/climate/climate-change-scheme.md (Adding explanatory text).

mukrop committed:
English text proofreading

mukrop commented:
Hello @willemfrishert! Sorry for a longer reply time.

In general, great job! A few minor comments and answers to your questions below.

### Your questions and content

* For the two preview boxes, there are really no English translations yet. For the moment, keep the preview box links commented out in the source code and when they'll exist in English, we'll uncomment them.
* Redirecting the link to Czech Wikipedia to point to the English equivalent page is exactly what was necessary to do, good thinking.
* There still are two internal links in the text you did not comment on (as they still point to slugs used in CZ site, they break the build – you should be able to see the problem also in the [Travis log](https://app.travis-ci.com/github/faktaoklimatu/web-en/builds/257085178)).
* Regarding the links from above: One links the infographic on O2 cycles that is not yet available in English. Again, let's just comment the link out, The second links to the map of global temperature change that is already [published in English](https://factsonclimate.org/infographics/map-temperature-change). Thus, just point the link to the English one. Doing these two things should fix the build.

### Other comments

* I was testing running [Grammarly](https://grammarly.com/) on the text and I was pleasantly surprised by the result! I happen to have a pro license, but the free one also had several relevant suggestions :-). This is the only change I did directly in your branch.
* To replicate tests locally (e.g. checking internal links), you can run `make check`. The first set of tests has to pass (internal links to pages and images), while the second one is just informative (that checks availability of all external links – insights can be gained from it but we cannot guarantee their availability).
* You can see that HTML comments (`<!--` and `-->`) cause an end of block and thus some small whitespace issues (there is unintended indentation in places where the preview box lies in code). To eliminate this, you can use the Jekyll comments `{%- comment -%} xxxxxx {%- endcomment -%}`. It's just a nitpick, definitely not a publication blocker.
* Since adding a supplementary text is a significant change, I'd suggest adjusting the `published` date in the header.

willemfrishert committed:
Redirect Temperature Change Map from /infografiky/mapa-zmeny-teploty to /infographics/map-temperature-change.

willemfrishert committed:
Comment out link for /infografiky/cykly-koncentrace-co2

willemfrishert committed:
Use comment/endcomment tags from Jekyll/Liquid in favor of html comment tag to avoid small whitespace issues.

willemfrishert committed:
Update published date.

willemfrishert commented:
Hi @mukrop, thanks for the feedback!

Thanks for the heads up on [Grammarly](https://grammarly.com/). I'll see if I can get a free copy installed as it'll be a good aid to review the translated texts. I appreciate the pass you did and changes you made on top using Grammerly on your side.

A few updates from my side based on your suggestions:
- I replicated the issues highlighted by Travis using `make check`. Great to have automation on this part.
  - I commented out the first link using html commenting tags;
  - I redirected the second link to [the map of global temperature change](https://factsonclimate.org/infographics/map-temperature-change);
- I replaced the HTML comment tags I used to comment out the Jekyll preview boxes by [Liquid template comment tags](https://shopify.github.io/liquid/tags/template/). I noticed that the [Liquid whitespace controls](https://shopify.github.io/liquid/basics/whitespace/) aren't needed. The paragraphs start to appear nicely connected so instead of `{%- comment -%} xxxxxx {%- endcomment -%}`, I used `{% comment %} xxxxxx {% endcomment %}`. Let me know if you want me to have another look at it in case I missed something?
- I updated the `published` date in the header of the `climate-change-scheme.md`

Hope you can have another look at reviewing these changes?

mukrop commented:
Hi @willemfrishert,
Great job, thanks for the adjustments. At the moment, I see no other changes necessary. Thus, I'm merging the pull request (the deploy is automatic, you should be able to see you work online in a matter of minutes).

(Indeed, the Liquid whitespace controls were not needed in this case – but I appreciate the effort to investigate the matter properly!)

As for further cooperation, I'll write options in the email :-).
